AnaClos.HabitTracker/AnaClos.HabitTracker/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "AnaClos.HabitTracker": {
+      "commandName": "Project",
+      "workingDirectory": "C:\\Users\\efalcom\\source\\repos\\CodeReviews.Console.HabitTracker\\AnaClos.HabitTracker\\AnaClos.HabitTracker"
```

Comment:
Remember that if you hard-code paths that do not exist on other peoples machines you need to give users some sort of clue to find and replace... #WorksOnMyMachine 😄 

This would be perfect information to put in a README.md file... 🤔